libraries/stdlib/src/kotlin/util/BigDecimals.kt

```diff
@@ -89,7 +89,7 @@ public inline operator fun BigDecimal.dec(): BigDecimal = this.subtract(BigDecim
  */
 @SinceKotlin("1.2")
 @kotlin.internal.InlineOnly
-public inline fun Int.toBigDecimal(): BigDecimal = BigDecimal(this)
+public inline fun Int.toBigDecimal(): BigDecimal = BigDecimal.valueOf(this)
```

Comment:
There's no such overload of `valueOf`. Please ensure it compiles.

Reply:
Sorry, fixed that.